run-tests.js

```diff
@@ -77,10 +79,20 @@ const exec = (bin, ...args) =>
         let hasWarnings = false
 
         ps.stdout.on ('data', data => { output += data.toString () })
-        ps.stderr.on ('data', data => { output += data.toString (); stderr += data.toString (); hasWarnings = true })
+        ps.stderr.on ('data', data => { 
+            const dataString = data.toString ();
+            if (!isDebugMode || !dataString.includes('ExperimentalWarning: The Fetch API is an experimental feature. This feature could change at any time')) {
```

Comment:
so, according to this line, current flow is not affected at all (`! isDebugMode`) and only if debug is being used, then the check happens for `dataString.includes...`